config/libvirt-provider-onmetal/libvirt-provider-rbac/role.yaml

```diff
@@ -0,0 +1,161 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: provider-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - compute.api.onmetal.de
+  resources:
+  - machines
+  verbs:
+  - create
+  - delete
```

Comment:
Does it need access to delete all the resources(Machines, etc) included in this Role?

Reply:
For most of the resources it's probably not needed but I would keep it for now in sync with the broker roles.